### PR TITLE
Reyes story

### PR DESCRIPTION
**Pull request by christophereyes**
Please review this request @casey-collab issues corrected.

### CODE REVIEW

**Comment by pmallory** [comment]
@casey-collab can you review this PR please?

File: english/texas/texas.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+The man informs you that you are in Texas!!
+No wonder everything is so big.
+Time to follow your [destiny](../destiny.destiny.md).
```
Comment (casey-collab) [inline english/texas/texas.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/texas/texas.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+Or just [alchemy](../alchemy/alchemy.md) time!
```
Comment (casey-collab) [inline english/texas/texas.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by christophereyes** [comment]
@casey-collab issues have been corrected, could you review my request please?

**Comment by casey-collab** [comment]
Unfortunately this pull request could not be automatically reviewed due to technical difficulty.

**Comment by christophereyes** [comment]
@casey-collab issues have been corrected, could you review my request please?

**Comment by casey-collab** [comment]
Unfortunately this pull request could not be automatically reviewed due to technical difficulty.

**Comment by christophereyes** [comment]
@pmallory could I get assistance getting my pull request reviewed?

**Comment by pmallory** [comment]
huh, this looks good to me 👍

**Pull request merged**